index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+
+<html Lang="en-GB">
+
+<head>
+	
+	<link rel="stylesheet" type= "text/css" href="styles.css">
+	
+	<title>Ollie Ritchie, Portfolio</title>
+	
+</head>
+
+<body>
+	
+	<nav>
+		<div class="nav-bar">
+			<a href="#">About</a>
+			<a href="#">Portfolio</a>
+			<a href="#">Contact</a>
+		</div>
+		<button class=cv-download-button>Download CV</button>
+	</nav>
+	
+	<header class="welcome"> <!-- section contaning h1 title and a full width image -->
```

Comment:
whilst I like the naming here I wonder if it's too general to be a class, perhaps heading-section would be better? Looks like it could easily get confused with header? Same for other section class names.